earth_enterprise/BUILD_RPMS.md

```diff
@@ -0,0 +1,23 @@
+# Installing via Package Managers
```

Comment:
Can we move this to the `rpms` directory, and rename it to just `BUILD.md` since it seems to apply to both RPMs and Debs?

Reply:
Style: I prefer lines that have no more than about 80 characters.  It makes it easier to comment on the just the part you're suggesting changes to.

Reply:
We certainly could move it there...the md editor I was using likes to automatically join paragraphs continually, but is easy enough to split it up the lines.

Reply:
Actually, given the existing BUILD_Ubuntu.md, and BUILD_RHEL_CentOS.md, it might be argued it should be left where it is and renamed BUILD_RPMS_DEBS.md, but since we are not supporting debs yet, I think it may make sense/could be more consistent to leave where it is for now.  So I do think either option I think is fine @tst-ppenev.  I do fully agree the style and the typo/usage suggestions should clearly be done though, for example, having centos consistently use CentOS in the text.

Reply:
We can leave it as is, then.